### ISSUE TITLE
想咨询一下，data/dictionary_and_semantic_hierarchy.txt有对应的中文标签吗？

### ISSUE THREAD

**Issue by wwfnwg** [Tencent/tencent-ml-images#17]
data/dictionary_and_semantic_hierarchy.txt有对应的中文标签吗？

**Comment by wubaoyuan** [comment]
@wwfnwg 暂时没有

**Comment by wwfnwg** [comment]
👌，多谢，另外想请教一下，我们这个项目的多标签预测，我是这样写的：
 logits = net.logit
 logits = tf.sigmoid(logits)
你们预测的时候也是这样吗？

**Comment by wwfnwg** [comment]
这个后期你们能提供一个模版吗？

**Comment by wubaoyuan** [comment]
@wwfnwg 我们训练也用的是多sigmoid. 细节请参考train.py

**Comment by manlinting** [comment]
求中文标签 +1

**Comment by chenzhengnan** [comment]
请问你imagenet都下载好了吗
